Dockerfile

```diff
@@ -43,4 +43,5 @@ RUN wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.z
 
 # Copy code and install app dependencies
 COPY . /usr/src/app/
-RUN bundle install
+# Run bundler install in parallel with the amount of available CPUs
+RUN bundle install --jobs="$(nproc)"
```

Comment:
nice one!
I think we can use this one in ofn-install (our ansible magic toolbox to manage our servers) here:
https://github.com/openfoodfoundation/ofn-install/blob/eb6ed5b449e5ad676ff9601648119bb9eacc6b13/roles/deploy/tasks/deploy.yml#L55